scripts/Metacalibration/metacal.py

```diff
@@ -0,0 +1,189 @@
+"""
+Implements the basic process of metacalibration.
+Calculates the R shear response matrix from a galaxy
+"""
+
+import galsim
+import numpy as np
+
+
+ 
+def generate_observed_galaxy(source_galaxy, psf_blur, lensing_g1, lensing_g2):
+	"""
+	Takes in a source_galaxy and psf_blur GalSim objects as well as the parameters
+	g1 and g2 of the lensing shear and generates a new GalSim object representing
+	an observed galaxy
+	"""
```

Comment:
STYLE COMMENT: I guess for a reader that don't know anything about GalSim, it might be hard to know what kind of parameters should be put inside. To have a more detailed doc, might be useful for future reader. I was thinking of something like that: 

```suggestion
	"""
 	Takes in a source_galaxy and psf_blur GalSim objects as well as the parameters
 	g1 and g2 of the lensing shear and generates a new GalSim object representing
 	an observed galaxy
 	
 	:source_galaxy: galsim object. describe what it is and give an example.
 	:psf_blur:           galsim object. describe what it is and give an example.
 	:lensing_g1:       float. describe what it is and give an example.
 	:lensing_g2:      float. describe what it is and give an example.
 	"""
```

Reply:
I will not put it for other functions, but I would recommend to do something similar